portal/src/atoms/Modal.jsx

```diff
@@ -88,6 +88,7 @@ export const Modal = ({
                 'md:w-screen-md md:max-h-[70vh]': size === 'md',
                 'md:w-screen-lg md:max-h-[80vh]': size === 'lg',
                 'md:w-screen': size === 'xl',
+                'md:w-screen-sm md:max-h-[80vh]': size === 'tall',
```

Comment:
I'm not a fan of adding a tall size option since it's a different dimensions than the other sizes. I think instead of this, we can just remove the max height for the different sizes. I don't think the max height was part of the design; I'm not really sure why I added it.